.github/workflows/ci.yml

```diff
@@ -19,17 +19,17 @@ jobs:
       matrix:
         rust: ["1.59.0", "stable"]
     steps:
-      - uses: hecrj/setup-rust-action@967aec96c6a27a0ce15c1dac3aaba332d60565e2
+      - uses: hecrj/setup-rust-action@50a120e4d34903c2c1383dec0e9b1d349a9cc2b1
         with:
           rust-version: ${{ matrix.rust }}
           components: rustfmt,clippy
       - uses: actions/checkout@v1
       - name: "Get cargo bin directory"
         id: cargo-bin-dir
-        run: echo "::set-output name=dir::$HOME/.cargo/bin"
+        run: echo "dir=$HOME/.cargo/bin" >> $GITHUB_OUTPUT
```

Comment:
note: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/